tests/console/firewalld.pm

```diff
@@ -20,7 +20,7 @@ use version_utils 'is_tumbleweed';
 # Check Service State, enable it if necessary, set default zone to public
 sub pre_test {
     record_info 'Check Service State';
-    assert_script_run("if systemctl is-active firewalld | grep -q -i 'inactive'; then systemctl start firewalld; fi");
+    assert_script_run("if ! systemctl is-active -q firewalld; then systemctl start firewalld; fi");
```

Comment:
and if you would just unconditionally run `systemctl start firewalld` ? you will get zero exit code in both case

Reply:
Could be done as well, I assumed there was a reason to only call it if necessary.

Reply:
@asmorodskyi, I corrected this issue on my branch by doing what you proposed. But as @Vogtinator has already fixed it, I won't as for a PR.